content/app-dev/create-actor/update.md

```diff
@@ -18,7 +18,7 @@ Now, let's start the actor again using the "Start Actor" dropdown, but this time
 make target-path-abs
 ```
 
-Copy this file location and input it in the _Start Actor_ form. You can start multiple replicas of the actor here, I've chosen to start with `15` so that I have a few replicas while I test.
+Copy the path printed by this command into the and input it in the _Start Actor_ form, input a number of replicas, and hit submit.
```

Comment:
I still don't like the verb `input` here - please change to `enter` or `type`.

Reply:
I thought I fixed that, either way it's corrected now.